<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServisDialog</class>
 <widget class="QDialog" name="ServisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Teknik Servis</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="DonanimBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Cihazla gelen donanımlar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_1">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Çanta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_2">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Cihaz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_3">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>1. batarya</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_4">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>2. batarya</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_5">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>SDI kablo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_6">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>HDMI kablo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_7">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Desktop Şarj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_8">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Adaptör</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_9">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>SIM kartlar(1-6)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_10">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>SIM kartlar slotları (1-6)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="donanim_11">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>CVBS kablo(Görüntü - Ses)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="2">
          <widget class="QGroupBox" name="TestBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="testSuresi">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Test Süresi</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="testSuresi_">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="UIDNoBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="UIDNo">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>UID No</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="UIDNo_">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="servisNoBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="servisNo">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Servis No</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="servisNo_">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="ArizaTarifiBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="arizaTanimi">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Arıza Tarifi</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="arizaTanimi_">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="SevkBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="sevkTarihi">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Sevk Tarihi</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="sevkTarihiDay_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sevkTarihiMonth_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sevkTarihiYear_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QGroupBox" name="YapilanIslemBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="yapilanIslem">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Yapılan İşlem</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yapilanIslem_">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="musteriAdiBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="musteriAdi">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Müşteri Adı</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="musteriAdi_">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="GelisBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="gelisTarihi">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Geliş Tarihi</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="gelisTarihiDay_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="gelisTarihiMonth_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="gelisTarihiYear_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="BitisTarihBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="bitisTarihi">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Tamir Bitiş Tarihi</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="bitisTarihiDay_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="bitisTarihiMonth_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="bitisTarihiYear_">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Servis Ekle</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
